### ISSUE TITLE
[Combobox] Tags in active comboboxes can be focused and are not read by screen readers

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3009]
## Current behavior 
All active comboboxes with tags: 
1. [Group Selected](https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--group-selected) 
2. [Groups With Group Selected](https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--groups-with-group-selected)
3. [Selected](https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--selected)
4. [Stacked](https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--stacked)
5. [Value](https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--value)
6. [Virtual List](https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--virtual-list) 
7. [Virtual List With Groups](https://storybook.sanomalearning.design/?path=/story/form-combobox-multiple--virtual-list-with-groups)

All active comboboxes with tags use `<sl-tag>` to present selected options. Each `<sl-tag>` has `aria-hidden='true'` and the first one in the list has `tabindex='0'` which make whole list focusable with arrow keys. While user's focus is on each tag its name is not being announced as the element is hidden with ARIA. 
`<sl-tag-list>` inside which tags are is not hidden with `aria-hidden='true'` and use can place screen reader there using arrow keys as well. 
Both `<sl-tag-list>` and `sl-tag` shouldn't be announced with assistive technologies as its user can use listbox to get information about what is selected.

https://github.com/user-attachments/assets/379ac732-5a30-4ea3-bd3d-eb23a4409303

https://github.com/user-attachments/assets/6c51f998-03bc-409e-970b-d0d8ca6b7508

## Proposed solution
1. `sl-tag-list` should be hidden with `aria-hidden='true'` so user won't place screen reader focus on that element.
 2. All `sl-tag` should have `tabindex='-1'` or no `tabindex` at all, so keyboard focus can't be placed there. 

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS team

**Comment by a11ymiko** [comment]
When tag list is present inside combobox user can set focus inside combobox and using left arrow move fake focus to whole tags. This is not announced by VoiceOver and NVDA announce each item as 'blank' then. When this fake focus is placed on any of tags user cannot browse the expanded list of options with arrows up and down. Browsing this list is only possible when focus is placed back on combobox input.

https://github.com/user-attachments/assets/eb04fb0b-90f5-43b4-b1af-4fdf273db91d

Once user place focus on combobox input with `Tab` or `Shift + Tab` arrows left and right should not move focus to taglist. Only when focus is placed on tag in tag list then arrows should be used to move between tags.

**Comment by a11ymiko** [comment]
Right now when user moves in tag list with arrows they hear 'Use arrow keys to move between removable tags' after each tag. This is unnecessary and makes navigation longer.

https://github.com/user-attachments/assets/0584b86d-6ca0-4b2d-8c94-8b7517673fab

Fix for that issue is to add `aria-hidden='true'` to `<span>` with `navigationDescription`. Then it still will be announced when user place their focus on close button of each tag but when user will be using different navigation modes there won't be that much repeating, not necessary elements with the same text. Hidden description works like that, because `aria-describedby` is announcing even the text that is hidden with `aria-hidden`. It works this way so it's possible to put description text anywhere on the page and not to worry about it being announced as plain text out of context.
